deploy/utils.sh

```diff
@@ -102,6 +102,8 @@ runDockerCommand() {
     -e OPENSHIFT_PASSWORD \
     -e DOCKER_REGISTRY_PATH \
     -e DOCKER_REGISTRY_URL \
+    -e PULL_DOCKER_REGISTRY_PATH \
+    -e PULL_DOCKER_REGISTRY_URL \
```

Comment:
@izgeri can you elaborate on the difference between these 2?

Reply:
I'm following a pattern we used in k8s-demo when we added 4.3/4.5 support [PR](https://github.com/conjurdemos/kubernetes-conjur-demo/pull/124/files) - maybe @doodlesbykumbi can elaborate?

Reply:
Looking at the table in the PR I linked:
- `DOCKER_REGISTRY_URL` is the Docker registry URL
- `DOCKER_REGISTRY_PATH` is `{DOCKER_REGISTRY_URL}/{path}` where `{path}` is any platform specific organization path
- `PULL_DOCKER_REGISTRY_URL` is the Docker registry URL you use for _pulling_ images - in OCP 4.x, we have to use an internal registry URL for pulling images or the image pull fails - so there's a different URL for pushing (which is done from your local machine or an executor, e.g. external to the cluster) vs pulling (internal to the cluster)
- `PULL_DOCKER_REGISTRY_PATH` is `PULL_DOCKER_REGISTRY_URL/{path}` where, as above, `{path}` is any platform specific organization path